src/Equinor.ProCoSys.IPO.Domain/AggregateModels/PersonAggregate/Person.cs

```diff
@@ -72,7 +72,7 @@ public void RemoveSavedFilter(SavedFilter savedFilter)
         }
 
         public SavedFilter GetDefaultFilter(Project project) =>
-            _savedFilters.SingleOrDefault(s => s.ProjectId == project.Id && s.DefaultFilter);
+            _savedFilters.SingleOrDefault(s => s.ProjectId == project?.Id && s.DefaultFilter);
```

Comment:
Add unittest in PersonTests where project is null

Reply:
Done in CreateSavedFilterWithoutProject_ShouldSaveFilter()